ui/views.py

```diff
@@ -136,6 +137,16 @@ def __call__(self, request, repo_slug):
         if repo_slug not in set([x.slug for x in repos]):
             return HttpResponseForbidden("unauthorized")
         self.repo = [x for x in repos if x.slug == repo_slug][0]
+        # get sorting from params if it's there
+        params = dict(request.GET.copy())
+        if ('sortby' in params and
```

Comment:
The `sortby` can just be popped here, which would make this `if` statement shorter, and ensure it wasn't there so the lines below that pop it just to make sure it isn't there can be deleted.